### PR TITLE
feat(style): Import aerial hybrid map style.

### PR DESCRIPTION
**Pull request by Wentao-Kuang**

### CODE REVIEW

File: config/style/aerialhybrid.json [hunk 1]
```diff
@@ -0,0 +1,1315 @@
...
+      "url": ""
+    },
+    "LINZ-Imagery": {
+      "type": "raster",
+      "tileSize": 256,
+      "tiles": [],
+      "minzoom": 0,
+      "maxzoom": 28
+    },
+    "Nextzen-DEM": {
```
Comment (blacha) [inline config/style/aerialhybrid.json:27]: I dont think this needed.

File: config/style/aerialhybrid.json [hunk 2]
```diff
@@ -0,0 +1,1315 @@
+{
+  "version": 8,
+  "name": "aerialhybrid",
+  "metadata": {
+    "maputnik:license": "https://github.com/maputnik/osm-liberty/blob/gh-pages/LICENSE.md",
+    "maputnik:renderer": "mbgljs",
+    "openmaptiles:version": "3.x"
+  },
+  "sources": {
+    "natural_earth_shaded_relief": {
```
Comment (blacha) [inline config/style/aerialhybrid.json:10]: remove

File: config/style/aerialhybrid.json [hunk 3]
```diff
@@ -0,0 +1,1315 @@
...
+      "maxzoom": 28
+    },
+    "Nextzen-DEM": {
+      "type": "raster-dem",
+      "tiles": ["https://s3.amazonaws.com/elevation-tiles-prod/terrarium/{z}/{x}/{y}.png"],
+      "minzoom": 0,
+      "maxzoom": 14
+    }
+  },
+  "sprite": "https://maputnik.github.io/osm-liberty/sprites/osm-liberty",
```
Comment (blacha) [inline config/style/aerialhybrid.json:34]: are we allowed to use these sprites/glyphs? do we use them elsewhere?

Comment (Wentao-Kuang) [reply config/style/aerialhybrid.json:34]: I am not sure about these, why these are here.

**Review by blacha — APPROVED** [review]

**Pull request merged**